Sources/Deboogger.swift

```diff
@@ -31,6 +31,16 @@ private final class AssistiveButtonPresenterViewController: UIViewController {
     }
 }
 
+public struct DebooggerGesture {
+    public let numberOfTouches: Int
+    public let numberOfTaps: Int
+
+    public init(numberOfTouches: Int = 2, numberOfTaps: Int = 2) {
```

Comment:
That's pretty questionable default gesture, in some apps it may be quite common. Why not use 4 / 2 like we did?

Reply:
It turned out that 4/2 gesture is tough for some members of our team. Also it is easy to configure gesture if there are conflicts with app functionality 🤔

Reply:
It's as easy to configure another gesture if default one is too hard. 
I'd rather minimize the chance of unwanted deboogger trigger first, then think about convenience - functionality conflict might not be obvious without a hindsight, while figuring out a more convenient gesture is pretty easy.